AzureCalling/Views/ParticipantView.swift

```diff
@@ -45,6 +46,10 @@ class ParticipantView: UIView {
         participantLabelView.updateMuteIndicator(isMuted: isMuted)
     }
 
+    func updateActiveSpeaker(isSpeaking: Bool) {
+        activeSpeakerView.isHidden = !isSpeaking
```

Comment:
Inside of adding an activeSpeakerViewLayer, could we just update the participantView.border and borderColor?

Reply:
I guess this one confuses me as to whether we manipulate UI logic in code or rely on the views in storyboard